### PR TITLE
Clarify behavior of only-return-existing with no existing

### PR DESCRIPTION
**Pull request by bifurcation**

### CODE REVIEW

**Comment by jsha** [comment]
Is 200 really correct? This seems like a 404 maybe? Also we should specify the body (empty).

**Comment by uhhhh2** [comment]
Would a problem document be appropriate (with either a pre-existing ACME error type or a new one like "accountDoesNotExist")?

**Comment by martinthomson** [comment]
404 talks about the effective request URI, so that's not going to work.

I think that a 400 would be better on the basis that the request checked, but failed (@uhhhh2's suggestion is a fine enhancement).  The reason you make this request is to check; and having a 4xx series code is easier to test than the absence of a header field.

**Comment by bifurcation** [comment]
Thanks @martinthomson and @uhhhh2, implemented this in the latest commit.

File: draft-ietf-acme-acme.md [hunk 1]
```diff
@@ -1007,8 +1008,8 @@ account to be created if one does not already exist, then it SHOULD do so by
 sending a POST request to the new-account URL with a JWS whose payload has an
 "only-return-existing" field set to "true" ({"only-return-existing": true}).
 If a client sends such a request and an account does not exist, then the server
-MUST return a response with a 200 (OK) status code and no Location header
-field.
+MUST return am error response with status code 400 (Bad Request) and type
```
Comment (uhhhh2) [inline draft-ietf-acme-acme.md]: "an error"

**Pull request merged**